### PR TITLE
able to set default options

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,15 +3,20 @@ const async = require('async');
 const Logr = require('logr');
 const path = require('path');
 const fs = require('fs');
+const defaults = require('lodash.defaults');
 const bytesize = require('bytesize');
 
+let DefaultOptions = {
```
Comment (jgallen23) [inline index.js]: No title case. Surprised lint didn't catch

File: index.js [hunk 2]
```diff
@@ -3,15 +3,20 @@ const async = require('async');
...
+  description: 'Default ClientKitTask Description',
```
Comment (jgallen23) [inline index.js]: Default to blank

File: index.js [hunk 3]
```diff
@@ -3,15 +3,20 @@ const async = require('async');
...
+};
+
 class ClientKitTask {
   constructor(name, options, runner) {
-    options = options || {};
     this.name = name;
-    this.options = options;
-    this.runner = runner;
-    this.log = new Logr({
+    this.options = defaults(options, DefaultOptions);
```
Comment (jgallen23) [inline index.js]: Defaults should be on the class. Idea is that specific tasks will have their own options and the task needs to be able to set it

File: index.js (hunk 1 above)
Comment (orthagonal) [inline index.js]: If I add a lint error like 'var x = 5' right under this line, then my linter correctly identifies all the problems with it.  So there may be something wrong with the capitalization rule in our lint config.

**Pull request merged**